.gitignore

```diff
@@ -56,3 +56,6 @@ testfixtures_shared/
 
 # These are generated from .ci/jobs.t
 .ci/jobs/
+
+# build files generated
+doc-tools/missing-doclet/bin/main/org/opensearch/missingdoclet/MissingDoclet.class
```

Comment:
Can we add the whole directory `doc-tools/missing-doclet/bin/` rather than a single file?

There will be multiple of them, for example:

```
/OpenSearch/doc-tools/missing-doclet/bin/main/org/opensearch/missingdoclet$ ls
'MissingDoclet$1.class'  'MissingDoclet$2.class'  'MissingDoclet$3.class'   MissingDoclet.class

Reply:
👍 to @VachaShah idea

Reply:
That's better actually. Pushed the change.